### DISCUSSION TITLE
Handle new size record banners

### DISCUSSION THREAD

**Discussion by frozen-impact** [jonaro00/AutoTrader#3] [Ideas]
Hey! Love the script. Any ideas on how to handle the new size record banner that sometime appears on the overlay? It uses the same x button coordinate so perhaps the script needs some sort of fall back to handle it if appears.

A temporary workaround is try to avoid tagging xxl and xxs Pokemon but that doesn't always guarantee a record will be broken or not post-trade.

**Comment by jonaro00** [comment]
Hi, glad the script helps you!

With the current architecture, handling the size record screen is not possible.

I have been considering starting development of a new and probably over-engineered solution that uses screenshots to handle everything. That type of solution would solve, this issue, but it might make the program fail more due to increased complexity.

This size record popup might be the final straw that gets me started on the idea. Will see if I get around to it.

**Reply by eflcl** [reply]
i completely have no idea on how to perform these actions written in your post, would you mind do a detailed guide, like a video :), so far ive only managed to install adb and added it into PATH & python 3.11
